cluster-operator/src/main/resources/kafka-2.1.0-config-model.json

```diff
@@ -0,0 +1,808 @@
+{
```

Comment:
We dropped 2.1.0, 2.1.1 and 2.2.0 already in master. Do we still need these files?